app/src/main/java/com/mayokunadeniyi/instantweather/ui/search/SearchResultAdapter.kt

```diff
@@ -12,7 +13,7 @@ import com.mayokunadeniyi.instantweather.databinding.ItemSearchResultBinding
  * Created by Mayokun Adeniyi on 28/04/2020.
  */
 
-class SearchResultAdapter(val clickListener: SearchResultListener) : PagedListAdapter<SearchResult,SearchResultAdapter.ViewHolder>(SearchResultDiffCallBack()){
+class SearchResultAdapter(val clickListener: SearchResultListener) : ListAdapter<SearchResult,SearchResultAdapter.ViewHolder>(SearchResultDiffCallBack()){
```

Comment:
Also, we want to utilize a [PagedListAdapter](https://developer.android.com/reference/androidx/paging/PagedListAdapter) to help load the search location suggestion result in chunks so as to not waste the user's bandwidth loading all the list.